pkg/pgmodel/ingestor/ingestor.go

```diff
@@ -31,7 +31,7 @@ type Cfg struct {
 
 // DBIngestor ingest the TimeSeries data into Timescale database.
 type DBIngestor struct {
-	sCache     cache.SeriesCache
+	SCache     cache.SeriesCache
```

Comment:
Since `SeriesCache` is passed to Ingestor I'd recommend passing it into Rule adapter as well. That would eliminate the need of making it public here.

Reply:
I want to keep the code in rules minimalistic. If we don't want to make it public, lets go with exposing a function to access ingestor's series cache.